Microsoft.HealthVault.Fhir/Transformers/PersonItemToFhir.cs

```diff
@@ -0,0 +1,170 @@
+
+using System;
+using System.Linq;
+using Hl7.Fhir.Model;
+using Microsoft.HealthVault.Fhir.Constants;
+using Microsoft.HealthVault.ItemTypes;
+
+namespace Microsoft.HealthVault.Fhir.Transformers
+{
+    public static partial class ItemBaseToFhir
+    {
+        public static Practitioner ToFhir(this PersonItem person)
+        {
+            return PersonItemToFhir.ToFhirInternal(person);
+        }
+    }
+    public class PersonItemToFhir
+    {
+        internal static Practitioner ToFhirInternal(PersonItem person)
+        {
+            var practitioner = new Practitioner();
+
+            HumanName fhirName = ToFhirInternal(person.Name);
+            practitioner.Name = new System.Collections.Generic.List<HumanName> { fhirName };
+
+            if (!string.IsNullOrEmpty(person.Organization))
+            {
+                var fhirAddress = new Hl7.Fhir.Model.Address
+                {
+                    Text = person.Organization,
+                    Use = Hl7.Fhir.Model.Address.AddressUse.Work
+                };
+                practitioner.Address.Add(fhirAddress);
+                //practitioner.AddExtension(HealthVaultExtensions.PersonOrganisation, new FhirString(person.Organization));
+            }
+
+            if (!string.IsNullOrEmpty(person.ProfessionalTraining))
+            {
+                var qualificationComponent = new Practitioner.QualificationComponent
+                {
+                    Code = new CodeableConcept
+                    {
+                        Text = person.ProfessionalTraining
+                    }
+                };
+                practitioner.Qualification = new System.Collections.Generic.List<Practitioner.QualificationComponent>
+                {
+                    qualificationComponent
+                };
+            }
+
+            if (!string.IsNullOrEmpty(person.PersonId))
+            {
+                practitioner.Identifier = new System.Collections.Generic.List<Identifier>
+                {
+                   new Identifier
+                   {
+                       Value = person.PersonId
+                   }
+                };
+            }
+
+            if (person.ContactInformation != null)
+            {
+                practitioner.Address.AddRange(
+                    person.ContactInformation.Address.Select(address => ToFhirInternal(address)));
+                practitioner.Telecom.AddRange(
+                    person.ContactInformation.Phone.Select(phone => ToFhirInternal(phone)));
+                practitioner.Telecom.AddRange(
+                    person.ContactInformation.Email.Select(email => ToFhirInternal(email)));
+            }
+
+            return practitioner;
+        }
+
+        private static HumanName ToFhirInternal(Name hvName)
```

Comment:
Can you abstract this out to where it can also be used in "Transformers/PersonalToFhir.cs"? There's very similar code there for doing HumanNames. We ended up deciding to just put the suffix and prefix as strings instead of doing anything with extensions and codes.